tasks.py

```diff
@@ -178,7 +179,7 @@ def deferred(self, *args, **kwargs):
 				raise errors.RequestTimeout()  # Task-API should retry
 		elif cmd == "unb":
 			if not funcPath in _deferedTasks:
-				logging.error("Ive missed a defered task! %s(%s,%s)" % (funcPath, str(args), str(kwargs)))
+				logging.error("Ive missed a deferred task! %s(%s,%s)", funcPath, args, kwargs)
```

Comment:
Who is Ive? ;-)

Reply:
?

Reply:
Idk. Maybe a friend of @tsteinruecken :)